recipes/list_items/contact_info/contact_info.vue

```diff
@@ -7,12 +7,13 @@
     class="dt-contact-info"
   >
     <template #left>
-      <div
+      <button
         v-if="showAvatar"
-        class="d-ps-relative"
+        class="d-ps-relative d-bgc-transparent d-baw0"
```

Comment:
It's kind of hard for a user to visually know this is a button.. do we need some kind of visual indication on hover? Let's at least add `d-c-pointer` here. Who was the designer for this change?

Reply:
I don't think we have a designer for this change, this requirement change comes from product team.